src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -109,6 +110,7 @@ override void OnLaunched(LaunchActivatedEventArgs e)
 
 				// Place the frame in the current Window
 				Windows.UI.Xaml.Window.Current.Content = rootFrame;
+				Console.WriteLine($"RootFrame: {rootFrame}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826543)